rosidl_parser/rosidl_parser/__init__.py

```diff
@@ -445,7 +445,7 @@ def parse_value_string(type_, value_string):
     if type_.is_primitive_type() and not type_.is_array:
         return parse_primitive_value_string(type_, value_string)
 
-    if type_.is_primitive_type() and type_.is_array and type_.type != 'string':
+    if type_.is_primitive_type() and type_.is_array:
```

Comment:
thanks for the comment, I was wondering what the original reason for this condition was.
reverted and added a todo